src/test/scala/org/broadinstitute/dsde/firecloud/service/WorkspaceApiServiceSpec.scala

```diff
@@ -412,10 +422,31 @@ class WorkspaceApiServiceSpec extends BaseServiceSpec with WorkspaceApiService w
       }
     }
 
-    "OK status is returned from POST on /workspaces/%s/%s/clone" in {
-      stubRawlsService(HttpMethods.POST, clonePath, OK)
-      Post(clonePath, WorkspaceCreate("namespace", "name", Map())) ~> dummyUserIdHeaders("1234") ~> sealRoute(workspaceRoutes) ~> check {
-        status should equal(OK)
+    "POST on /workspaces/.../.../clone for 'not protected' workspace sends non-realm WorkspaceRequest to Rawls and passes back the Rawls status and body" in {
+      val response = Workspace("namespace", "name", None, "foo", "bar", DateTime.now(), DateTime.now(), "bob", Map(), Map(), Map())
+      stubRawlsService(HttpMethods.POST, clonePath, Created, Some(response.toJson.compactPrint))
+
+      val orchestrationRequest: WorkspaceCreate = WorkspaceCreate("namespace", "name", Map())
+      val rawlsRequest: WorkspaceRequest = WorkspaceRequest("namespace", "name", None, Map(AttributeName("library", "published") -> AttributeBoolean(false)))
+      Post(clonePath, orchestrationRequest) ~> dummyUserIdHeaders("1234") ~> sealRoute(workspaceRoutes) ~> check {
+        rawlsServer.verify(request().withPath(clonePath).withMethod("POST").withBody(rawlsRequest.toJson.prettyPrint))
+        status should equal(Created)
+        responseAs[Workspace] should equal(response)
+      }
+    }
+
+    "POST on /workspaces/.../.../clone for 'protected' workspace sends dbGap-realm WorkspaceRequest to Rawls and passes back the Rawls status and body" in {
+      val realm: Option[RawlsRealmRef] = Option(RawlsRealmRef(RawlsGroupName("dbGapAuthorizedUsers")))
+      val response = Workspace("namespace", "name", realm, "foo", "bar", DateTime.now(), DateTime.now(), "bob", Map(), Map(), Map())
+      stubRawlsService(HttpMethods.POST, clonePath, Created, Some(response.toJson.compactPrint))
+
+      val isProtected: Option[Boolean] = Option(true)
+      val orchestrationRequest: WorkspaceCreate = WorkspaceCreate("namespace", "name", Map(), isProtected)
+      val rawlsRequest = WorkspaceRequest("namespace", "name", realm, Map(AttributeName("library", "published") -> AttributeBoolean(false)))
+      Post(clonePath, orchestrationRequest) ~> dummyUserIdHeaders("1234") ~> sealRoute(workspaceRoutes) ~> check {
+        rawlsServer.verify(request().withPath(clonePath).withMethod("POST").withBody(rawlsRequest.toJson.prettyPrint))
+        status should equal(Created)
+        responseAs[Workspace] should equal(response)
```

Comment:
From slack discussion: `rawlsRequest` should start off with `AttributeName("library", "published") -> AttributeBoolean(true)`, and then that will be transformed inside the clone endpoint. At that point, the rawls stub should see `AttributeName("library", "published") -> AttributeBoolean(false)` instead.